examples/node/README.md

```diff
@@ -0,0 +1,24 @@
+# Node demo
+
+## Requirements
+
+Build the base project `/meilisearch-js` as this example uses the locally build bundles from `meilisearch-js` 
+
+```
+$ cd meilisearch-js
+$ yarn build
+```
+
+## Credentials 
+
+This example uses the following MeiliSearch address: `http://127.0.0.1:7700`. Feel free to change the credentials to meet your MeiliSearch address.
+
+The credentials are written in each script.
+
+## Try out
+
+To try it out you just need to execute the files: 
```

Comment:
```suggestion
To try it out you just need to execute the file:
```

Reply:
It's files because their are more than one file in this directory,

Reply:
But as the command below is `node search_example.js`, this is one file, right?

Reply:
It is one of the files that are present. Maybe I should add `For example: `

Reply:
As you wish

Reply:
done